packages/core/package.json

```diff
@@ -99,33 +95,8 @@
     "cypress-axe": "^1.5.0",
     "cypress-wait-until": "^2.0.1",
     "dotenv": "^8.2.0",
-    "eslint": "^7.32.0",
-    "husky": "^5.2.0",
     "is-ci": "^3.0.0",
     "jest": "^29.7.0",
-    "lint-staged": "^15.2.0",
-    "stylelint": "^14.6.0",
-    "stylelint-config-recess-order": "^3.0.0",
-    "stylelint-config-standard": "^24.0.0",
-    "stylelint-config-standard-scss": "^3.0.0",
-    "stylelint-order": "^5.0.0",
-    "stylelint-scss": "^4.0.1",
     "ts-jest": "29.1.1"
-  },
-  "lint-staged": {
-    "*.{ts,js,tsx,jsx}": [
-      "eslint --fix",
-      "prettier --write"
-    ],
-    "*.json": [
-      "prettier --write"
-    ],
-    "*.scss": [
-      "stylelint --fix"
-    ]
-  },
-  "volta": {
-    "node": "18.19.0",
-    "yarn": "1.19.1"
   }
```

Comment:
This is important to avoid package manager lock-in

Reply:
So Volta will not be used anymore? If so, how will we keep ensuring those versions after that?

Reply:
I though just declaring volta on the root package would be enough. Checked the documentation, and it is not: https://docs.volta.sh/advanced/workspaces. Added it again